rskj-core/src/test/java/co/rsk/peg/BridgeTestIntegration.java

```diff
@@ -2887,7 +2858,12 @@ public void testCallFromContract_beforeOrchid() {
 //        config.setBlockchainConfig(mockedConfig);
         blockFactory = new BlockFactory(config.getActivationConfig());
 
-        PrecompiledContracts precompiledContracts = new PrecompiledContracts(config, null);
+        BridgeSupportFactory bridgeSupportFactory = new BridgeSupportFactory(
+                new RepositoryBtcBlockStoreWithCache.Factory(constants.getBridgeConstants().getBtcParams()),
+                constants.getBridgeConstants(),
+                activationConfig);
+
+        PrecompiledContracts precompiledContracts = new PrecompiledContracts(config, bridgeSupportFactory);
```

Comment:
did it fail if you didn't pass a `BridgeSupportFactory`?

Reply:
Just tested and it is not failing. I don't recall why I add it back then TBH, will undo it.